config/ci-tests/verify_cuda_n2k.yaml

```diff
@@ -54,18 +62,42 @@ host_rfi_RFImask_buffer:
     frame_size: num_local_freq * samples_per_data_set / 8
     metadata_pool: main_pool
 
+host_pl_mask_buffer:
+    kotekan_buffer: standard
+    num_frames: buffer_depth
+    frame_size: ((samples_per_data_set / 2) * (num_elements / 8) * (num_local_freq / 4)) / 8
+    metadata_pool: main_pool
+
 host_correlation_buffer:
     kotekan_buffer: standard
     num_frames: buffer_depth
     frame_size: num_local_freq * (samples_per_data_set / sub_integration_ntime) * 16 * 16 * num_vis_blocks * 2 * sizeof_int
     metadata_pool: main_pool
 
+host_counts_buffer:
+    kotekan_buffer: standard
+    num_frames: buffer_depth
+    frame_size: (samples_per_data_set / sub_integration_ntime) * num_local_freq * counts_num_blocks * counts_blocksize * counts_blocksize * sizeof_int
```

Comment:
The `frame_size` here and for `host_correlation_buffer` should probably look equivalent. Maybe rearrange them so that they look more similar? Could also change the variable names to be more similar (`num_vis_blocks` vs `counts_num_blocks` etc.) Should either both hard-code `16` and `8`, or have constants for both.